anvio/kegg.py

```diff
@@ -5232,20 +5527,106 @@ def create(self):
                               "'ANNOTATION_SOURCE' fields to those module files, and then re-do this setup.")
         annotation_source_list = ",".join(list(self.annotation_sources))
 
+        if self.brite_dict:
```

Comment:
instead of such a long nested if, this part could go into its own function, e.g., `populate_brite_table`, and it would return with a warning early on `if not self.brite_dict`. It would also enforce a healthy level of modularity within the class.